cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -109,19 +108,54 @@ Downloading and Distributing Packages
 
 Downloading and Installing CSD
 ------------------------------
-- Download the CDAP CSD (Custom Service Descriptor) by `downloading the JAR file 
-  <http://cask.co/resources/#cdap-integrations>`__.
+To install CDAP on a cluster managed by Cloudera, we have available a Custom Service
+Descriptor (CSD) which you can install onto your CM server. This adds CDAP to the list of
+available services which CM can install. 
+
+.. _cloudera-compatibility-matrix:
+
++----------------------------------------------------------------------------------------------------------------------+
+| Supported Cloudera Manager (CM) and Cloudera Data Hub (CDH) Distributions                                            |
++------------+---------------------+---------------------+----------------------------+--------------------------------+
+| CM Version | CDH Versions        | CSD Versions        |  CDAP Parcel               | CDAP HBase Coprocessor Version |
++============+=====================+=====================+============================+================================+
+| 5.5        | 5.5                 | 3.3.x               | *Matching CSD major.minor* | ``hbase10cdh550``              |
++------------+---------------------+---------------------+----------------------------+--------------------------------+
+| 5.5        | less than 5.5       | 3.0.x through 3.2.x | *Matching CSD major.minor* | ``hbase10cdh``                 |
```

Comment:
The Coprocessor Version is tied to CDH version.  So this column is inaccurate, and I don't think it adds much value here anyway.  I'd suggest removing it from this table and have a separate table in the upgrade section of simply CDH Version => Coprocessor Version